<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1291</width>
    <height>808</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxTree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>ontology tree</string>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>791</width>
       <height>511</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxEditor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>ontology editor</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushAddSubclass">
        <property name="text">
         <string>add subclass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushAddClass">
        <property name="text">
         <string>add class</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushRemoveNode">
        <property name="text">
         <string>remove node</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="currentText">
        <string>&lt;select&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>&lt;select&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>load ontology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>visualize ontology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>create ontology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>save ontology</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="elucidationLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>selection elucidation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAddElucidation">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>460</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add changes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
